gvm/protocols/gmpv8/gmpv8.py

```diff
@@ -925,55 +925,6 @@ def create_filter(
 
         return self._send_xml_command(cmd)
 
-    def modify_filter(
```

Comment:
The change should have been done in another PR. It is very difficult to review PRs that are getting bigger and mixing several purposes.